slices/fonts-urw-base35.yaml

```diff
@@ -0,0 +1,23 @@
+package: fonts-urw-base35
+
+essential:
+  - fonts-urw-base35_copyright
+
+slices:
+  config:
+    contents:
+      /etc/fonts/conf.d/61-urw*:
+      /usr/share/fontconfig/conf.avail/urw-*:
+
+  fonts:
+    essential:
+      - xfonts-utils_bins
```

Comment:
`fonts` shouldn't need this dependency, not at least without a mutation script. `xfonts-utils` is used by this package's maintainer scripts, which are not reflected in this slice, so either 1) remove the dep (if the postinst is not needed), or 2) keep the dependency and add the mutation scripts

Reply:
I will test without mutation script first and see if the tests are fine.